tests/backends/test_vanilla_django_backend.py

```diff
@@ -164,6 +164,38 @@ def test_get_email_message_genrated_plain_text(self, mock):
 
     @patch.object(
         template_backend_klass, '_render_email',
```

Comment:
The whole code you wrote lives inside the `_render_email` method. If you mock it, it won't run.